app/api/contests/enter.robustness.feature

```diff
@@ -93,6 +85,10 @@ Feature: Enters a contest as a group (user self or team) (contestEnter) - robust
     Given the database has the following table 'items':
       | id |
       | 50 |
+    And the database has the following table 'permissions_generated':
```

Comment:
For all these, would it better to put them in `permissions_granted` and just let the propagation create these results. Otherwise you have quite unrealistic data with permission but nothing in `permissions_granted`

Reply:
These tests do not depend on `permissions_granted` at all, so it's must easier to seed the DB without thinking about the propagation rules.